specs/core/1_custody-game.md

```diff
@@ -168,8 +166,6 @@ This document details the beacon chain additions and changes in Phase 1 of Ether
     'revealer_index': ValidatorIndex,
     'period': 'uint64',
     'key': BLSSignature,
-    'masker_index': ValidatorIndex,
```

Comment:
Don't we want to completely nuke `CustodyKeyReveal`? That is, do all key reveals (early or not) with the same operation type.

Reply:
I'd support that. Though if so, `verify_custody_key` would need to be removed, and instead phase 1 would have an instruction to change how `process_randao_reveal_slashing` works.

Reply:
Integrated this